index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="main-container">
+            <div class="header-container">
+              <nav class="header">
+                <div class="my-logo">
+                  <a class="Logo" href="#">My Logo</a>
+                </div>
+                <div class="hambuger">
+                  <div class="bar"></div>
+                  <div class="bar"></div>
+                  <div class="bar"></div>
+                </div>
+              </nav>
+            </div>
+        </div>
+    </header>
```

Comment:
- Just like the previous reviewer requested, the margins in your navigations aren't equal. Please try to fix it, i Have pointed out the wrong margins in your design 🙏 🙏 

Your design looks like this currently:
![edited nav](https://user-images.githubusercontent.com/73607512/191862864-7df09f2e-2127-43c3-beb5-7c36319ecb4e.png)